main.css

```diff
@@ -0,0 +1,107 @@
+.header-container {
+  display: flex;
+  justify-content: space-between;
+  width: 375px;
+  height: 92px;
+}
+
+.header-container > nav {
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.nav-item1 {
+  flex-grow: 1;
+}
+
+.nav-item2 {
+  flex-grow: 8;
+}
+
+.nav-item3 {
+  flex-grow: 1;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  background-image: url("images/header.png");
+  background-size: 100%;
+  background-repeat: no-repeat;
+  width: 375px;
+  height: 604px;
```

Comment:
- [ ] Great job using flexbox. However, by giving a width it's affecting the responsiveness of the website by creating unnecessary space when the width is more than 375px I suggest removing the fixed width or giving max-width.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/158662107-c7af4d5a-596c-4245-a914-fdc7e5b756e2.png)